### ISSUE TITLE
WASM / Xamarin Forms - ListView, TableView doesn't work

### ISSUE THREAD

**Issue by zbigniew-gajewski** [praeclarum/Ooui#142]
After adding ListView or TableView to the page, the browser shows blank page and console shows error:

![image](https://user-images.githubusercontent.com/32226444/39098457-962a1dd6-466b-11e8-94a6-3421342ca464.png)

**Comment by praeclarum** [comment]
Wow, that's bad. Would you confirm that this is Ooui 0.10?

**Comment by zbigniew-gajewski** [comment]
Yes, this is 1.10.222 (Ooui, Ooui.Wasm, Ooui.Forms). Here is working example:
https://github.com/zbigniew-gajewski/ooui-wasm-aspnet-core-xaml

If you add **`<ListView></ListView>`** element just after Button element in
**OouiTestXamlApp/Views/MainPage.xaml** then this bug appears. 

Grid, Button, Label - all work OK.

**Comment by lucwuyts** [comment]
I have the same problem...

**Comment by davidlsharp1** [comment]
How's it looking on this one?  I just hit this as well.

**Comment by praeclarum** [comment]
Sorry for the delays. I’ll take a look at this tomorrow.

**Comment by snow-jallen** [comment]
Any news on this?  I believe I'm bumping into the same problem.
